opts/options.go

```diff
@@ -69,6 +71,13 @@ var Defaults = func(o *Options) error {
 	return nil
 }
 
+func RoutingTableLatencyTolerance(latency time.Duration) Option {
```

Comment:
Could you add some documentation for this function ?

Reply:
Sure.